src/SDKs/Azure.Base/data-plane/Azure.Base.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using Azure.Base.Attributes;
+
+[assembly: AzureSdkComponent("AzureSDK.Tests")]
```

Comment:
Also, did you see the syntax of UA header we want? It's described in general guidelines I forwarded you. I am myself not sure what's the algorithm to produce such syntax from our assebmly name. i.e. do we just remove all the dots and lowercase?

Reply:
We can only put the `client_lib` part into `AzureSdkClientLibraryAttribute` so it becomes `[AzureSdkClientLibrary("blobstorage")]`